keyboards/draytronics/elise/readme.md

```diff
@@ -0,0 +1,25 @@
+# ELISE
+
+![elise](https://www.draytronics.co.uk/wp-content/uploads/2021/04/repository-open-graph-template.png)
+
+A open source, low cost 65% keyboard PCB with USB-C, ESD protection, RGB underglow and XD68 case / plate compatability.  Designed in the UK. 
+More info / sales / PCB designs available at [draytronics.co.uk/elise](https://draytronics.co.uk)
+
+* Keyboard Maintainer: [Blake Drayson](https://github.com/ghostseven)
+* Hardware Supported: ELISE PCB / ATMega32U4
+* Hardware Availability: [draytronics.co.uk](https://draytronics.co.uk)
+
+Make example for this keyboard (after setting up your build environment):
```

Comment:
Could you add instructions on how to enter the bootloader to flash new firmware?

Reply:
Instructions providing various methods to enter DFU mode have now been added to the readme.md